ci/checks/enforce-issue-number-for-todos.sh

```diff
@@ -1,24 +1,7 @@
 #!/usr/bin/env bash
 
-function error() {
-  red='\033[91m'
-  endc='\033[0m'
-
-  if [[ $# -ge 1 ]]; then
-    if [[ -t 1 ]]; then
-      echo -e "${red}>>> $* <<<${endc}"
-    else
-      echo -e ">>> $* <<<"
-    fi
-  fi
-}
-
-function die() {
-  echo
-  error "$@"
-  echo
-  exit 1
-}
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
```

Comment:
Rel, #484 (don't do now, that's a good first issue for the next generation of contributors)